admin-data-explorer-api/shared/src/main/scala/net/wiringbits/webapp/utils/api/AdminDataExplorerApiClient.scala

```diff
@@ -11,13 +11,20 @@ import scala.util.{Failure, Success, Try}
 trait AdminDataExplorerApiClient {
   def getTables: Future[AdminGetTables.Response]
 
-  def getTableMetadata(tableName: String, offset: Int, limit: Int): Future[AdminGetTableMetadata.Response]
+  def getTableMetadata(
+      tableName: String,
+      sort: List[String],
+      range: List[Int],
+      filter: String
+  ): Future[List[Map[String, String]]]
 
-  def viewItem(tableName: String, id: String): Future[AdminFindTable.Response]
+  def viewItem(tableName: String, id: String): Future[Map[String, String]]
```

Comment:
Can we still have the response as `AdminFindTableItem.Response` which has a map in the result?